doc/api/string_decoder.md

```diff
@@ -82,3 +82,13 @@ Returns a decoded string, ensuring that any incomplete multibyte characters at
 the end of the `Buffer` are omitted from the returned string and stored in an
 internal buffer for the next call to `stringDecoder.write()` or
 `stringDecoder.end()`.
+
+### stringDecoder.reset([encoding])
```

Comment:
A nit: it seems this should go before the `stringDecoder.write()`, ABC-wise.

Reply:
Done!